core/src/main/java/org/apache/accumulo/core/client/admin/NewTableConfiguration.java

```diff
@@ -134,6 +149,13 @@ public NewTableConfiguration setProperties(Map<String,String> props) {
     if (limitVersion)
       propertyMap.putAll(IteratorUtil.generateInitialTableProperties(limitVersion));
 
+    if (createOffline)
+      propertyMap.put(Property.TABLE_OFFLINE_OPTS + "create.offline", "true");
+
+    if (createInitialSplits) {
+      propertyMap.put(Property.TABLE_OFFLINE_OPTS + "create.initial.splits", "true");
+    }
```

Comment:
Along with what Keith said about the Properties, these map keys should probably just be internal constants. since it looks like these actions are just performed through the API and not something you would set in the config.